package.json

```diff
@@ -14,6 +14,7 @@
     "thunderbird": ">=68.0a1"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "^1.1.0",
```

Comment:
Generally I prefer exact versions here so that we know when these are getting bumped.

Reply:
I'll change this :-)